platform/bungeecord/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/lokka30/Treasury>.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>treasury-parent</artifactId>
+        <groupId>me.lokka30</groupId>
+        <version>1.1.2-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>treasury-bungeecord</artifactId>
+
+    <name>Treasury-BungeeCord</name>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.2.4</version>
+                <configuration>
+                    <createDependencyReducedPom>false</createDependencyReducedPom>
+                    <relocations>
+                        <relocation>
+                            <pattern>org.bstats</pattern>
+                            <shadedPattern>me.lokka30.treasury.plugin.bungeecord.shade.bstats</shadedPattern>
+                        </relocation>
+                        <relocation>
+                            <pattern>com.mrivanplays.annotationconfig</pattern>
+                            <shadedPattern>me.lokka30.treasury.plugin.shade.annotationconfig</shadedPattern>
+                        </relocation>
+                        <relocation>
+                            <pattern>org.jetbrains</pattern>
+                            <shadedPattern>me.lokka30.treasury.plugin.bungeecord.shade.jetbrains</shadedPattern>
```

Comment:
There's no need to include the annotations classes at all, they do nothing at runtime. The downside is that they do have to be declared in every module as a dependency then, but that becomes a lot easier with use of the `dependencyManagement` system.

Doesn't really matter to me which route we take, but since some of us have expressed concerns about file size before it's food for thought.

Reply:
> There's no need to include the annotations classes at all, they do nothing at runtime. The downside is that they do have to be declared in every module as a dependency then, but that becomes a lot easier with use of the `dependencyManagement` system.
> 
> Doesn't really matter to me which route we take, but since some of us have expressed concerns about file size before it's food for thought.

A small compromise for a lower file size. Let's not shade it